appservice/src/main/resources/static/swagger/swagger-spec.yaml

```diff
@@ -1925,213 +2008,204 @@ definitions:
-        properties:
-          size:
-            type: string
-            description: The number of elements on the current page
-          totalElements:
-            type: string
-            description: The total number of elements
-            example: 300
-          totalPages:
-            type: string
-            description: The total number of pages
-            example: 15
-          number:
-            type: string
-            description: The current page number.
-            example: 1
-        required:
-          - size
-          - totalElements
-          - totalPages
-          - number
-    required:
-      - _links
-      - _embedded
-      - page
-  DayActivityOverview:
-    type: object
-    properties:
       _links:
         allOf:
           - $ref: '#/definitions/Curies'
           - $ref: '#/definitions/SelfLink'
-      date:
-        type: string
-        example: 2016-03-16
-        description: The day date in ISO 8601 format (yyyy-MM-dd)
-      _embedded:
-        properties:
-          "yona:dayActivities":
-            type: array
-            items:
-              $ref: '#/definitions/DayActivity'
+          - $ref: '#/definitions/NextLink'
+          - $ref: '#/definitions/PrevLink'
+          - $ref: '#/definitions/FirstLink'
+          - $ref: '#/definitions/LastLink'
+        required:
+          - curies
+          - self
     required:
-      - _links
-      - date
       - _embedded
-  DayActivity:
+      - page
+      - _links
+  DayActivityOverview:
     type: object
     properties:
-      _links:
-        allOf:
-          - $ref: '#/definitions/Curies'
-          - $ref: '#/definitions/SelfLink'
-        properties:
-          "yona:goal":
-            type: object
-            properties:
-              href:
-                type: string
-            required:
-              - href
-            description: Link to the goal for which activity was recorded. Only included in day overview or day detail because in other cases this information is already present on the week activity parent.
       date:
         type: string
+        format: date
         example: 2016-03-16
-        description: The day date in ISO 8601 format (yyyy-MM-dd). Only included in day detail because in other cases this information is already present on the parent.
+        description: The day date in ISO 8601 format (yyyy-MM-dd)
       timeZoneId:
         type: string
         description: The time zone identifier that the user had configured when this day started.
         example: Europe/Amsterdam
+      dayActivities:
+        type: array
+        items:
+          $ref: '#/definitions/DayGoalActivityOverview'
+      _links:
+        allOf:
+          - $ref: '#/definitions/Curies'
+          - $ref: '#/definitions/SelfLink'
+        description: The self link is currently not returned from the server. See YD-203
+    required:
+      - date
+      - dayActivities
+      - _links
+  DayGoalActivityBase:
```

Comment:
OK accepted, I think it is necessary after all to make resource distinction between the items in overview resources and the detail resources.